python/setup.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+import subprocess
+
+import setuptools
+import setuptools.command.build_py
+import setuptools.command.install
+import setuptools.command.develop
+import setuptools.command.test
+
+_this_dir = os.path.dirname(os.path.abspath(__file__))
+_clib_dir = os.path.abspath(os.path.join(_this_dir, 'opencc', 'clib'))
+_opencc_rootdir = os.path.abspath(os.path.join(_this_dir, '..'))
+
+assert os.path.isfile(os.path.join(_opencc_rootdir, 'CMakeLists.txt'))
+
+_build_dir = build_dir = os.path.join(_opencc_rootdir, 'build', 'python')
+_libopenccfile = os.path.join(_clib_dir, 'lib', 'libopencc.so')
+
+
+def _build_libopencc():
```

Comment:
Can you make this optional if libopencc is already installed?

Reply:
Yup will fix this!

Reply:
Okay I've changed this to checking if `opencc/clib/libopencc.so` exists.
It does not use the system installed version of `opencc` but I think it's better if for python packages to be self-contained if they depend on c binaries.